libs/dh/admin/feature-user-management/src/lib/feature-create-userrole/tabs/dh-create-userrole-tabs.component.html

```diff
@@ -0,0 +1,52 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let t; read: 'admin.userManagement.createrole'">
+  <ng-container *rxLet="validation$ as validation">
+    <watt-validation-message
+      type="danger"
+      *ngIf="validation"
+      [label]="t('errorToastLabel')"
+      [message]="validation!.error"
+    ></watt-validation-message>
+  </ng-container>
+
+  <watt-card *rxLet="isLoading$ as isLoading">
+    <watt-card-title>
+      <h4>{{ t("headline") }}</h4>
+    </watt-card-title>
+    <div class="spinner-container" *ngIf="isLoading">
+      <watt-spinner></watt-spinner>
```

Comment:
question: is a loading spinner necessary when a loading toast is shown while the form is submitting?

Reply:
Probably not, I'll remove it